1-getting-started-lessons/1-intro-to-programming-languages/README.md

```diff
@@ -78,36 +78,36 @@ Believe it or not, *they're both doing the same thing*: printing a Fibonacci seq
+A **development environment** is a unique set of tools and features that a developer uses often when writing software. Some of these tools have been customized for a developers specific needs, and may change over time if that developer changes priorities in work, personal projects, or when they use a different programming language. Development environments are as unique as the developers who use them.
 
 ### Editors
 
-One of the most crucial tools for software development is the editor. Editors are where you write your code and sometimes where you will run your code.
+One of the most crucial tools for software development is the editor. Editors are where you write your code and sometimes where you run your code.
 
 Developers rely on editors for a few additional reasons:
 
-- *Debugging* Discovering bugs and errors by stepping through code, line by line. Some editors have debugging capabilities, or can be customized and added for specific programming languages.  
-- *Syntax highlighting* Adds colors and text formatting to code, makes it easier to read. Most editors allow customized syntax highlighting.
-- *Extensions and Integrations* Additions that are specialized for developers, by developers, for access to additional tools that aren't built into the base editor. For example, many developers also need a way to document their code and explain how it works and will install a spell check extension to check for typos. Most of these additions are intended for use within a specific editor, and most editors come with a way to search for available extensions.
-- *Customization* Most editors are extremely customizable, and each developer will have their own unique development environment that suits their needs. Many also allow developers to create their own extensions.
+- *Debugging* helps uncover bugs and errors by stepping through the code, line by line. Some editors have debugging capabilities, they can be customized and added for specific programming languages.  
+- *Syntax highlighting* adds colors and text formatting to code, making it easier to read. Most editors allow customized syntax highlighting.
+- *Extensions and Integrations* are specialized tools for developers, by developers. These tools weren't built into the base editor. For example, many developers document their code to explain how it works. They may install a spell check extension to help find typos within the documentation. Most extensions are intended for use within a specific editor, and most editors come with a way to search for available extensions.
+- *Customization* enables developers to create a unique development environment to suit their needs. Most editors are extremely customizable and may also allow developers to create custom extensions.
```

Comment:
I don't think this is really necessary but will accept. We have to be careful not to deviate too much as these propagate to the translations